.github/workflows/phpunit-l8.yml

```diff
@@ -16,11 +16,13 @@ jobs:
             fail-fast: false
             matrix:
                 php: [8.0, 7.4, 7.3]
-                laravel: [8.*]
+                laravel: [9.*, 8.*]
```

Comment:
If you plan to drop Laravel 8 support you may streamline this build matrix to just PHP 8, Laravel 9, and Testbench 7.